_posts/2021-01-10-boa-release-11.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Boa release v0.11"
+author: Boa Developers
+---
+
+Boa has reached a new release. v0.11, our biggest one yet. Since v0.10 we've closed 77 issues, merged 129 pull requests. The engine has been faster and more compliant to the spec. Below are some of the highlights but please see the [changelog link] for more information.
+
+## Test 262
+
+Test262 is the implementation conformance test suite maintained by TC39. It's used by nearly all engines to measure how conformant they are to the specification. Boa pulls the tests in-tree and runs against PRs and on a nightly basis. Test262 doesn't sit still, so since the last release we've updated our reference to make sure we're running against the latest spec. You can find more about Test262 [here](https://github.com/tc39/test262).
+
+Since v0.10 we have almost doubled on spec conformance, and reduced panics from hundreds to less than 50. A year ago Boa didn't even track itself against Test262 so it was difficult to know how compliant we were to the spec, today not only do we track all changes against Test262 but we can see progress on a PR to PR basis.
```

Comment:
```suggestion
Since v0.10 we have almost doubled on spec conformance, and reduced panics from hundreds to 25 (tracked in [#817](https://github.com/boa-dev/boa/issues/817)). A year ago Boa didn't even track itself against Test262 so it was difficult to know how compliant we were to the spec, today not only do we track all changes against Test262 but we can see progress on a PR to PR basis.
```